src/js/tabular_comparison/tabular-comparison.module.css

```diff
@@ -0,0 +1,229 @@
+body {
```

Comment:
It will be good if we can use : https://mui.com/system/styled/
and remove css from module.css
also i think shouldn't it go in main css folder and attached to html file directly as it contain styles for body 🤔

Reply:
for now we decided to leave it like this